File: Faltantes o por revisar/EuclideanDistance.java
//Halla la distancia euclideana de 2 puntos en dos dimensiones (x,y). Para usar el primer método, debe definirse previamente la clase Point

import java.awt.Point;

public class EuclideanDistance {

    /*Trabajando con la clase Point*/
    static double euclideanDistance(Point p1, Point p2) {
        return Math.hypot(p1.x - p2.x, p1.y - p2.y);
    }

    /*Trabajando con los valores x y y de cada punto*/
    static double euclideanDistance(double x1, double y1, double x2, double y2) {
        return Math.hypot(x2 - x1, y2 - y1);
    }

    public static void main(String[] args) {
        System.out.println(euclideanDistance(new Point(0, 0), new Point(20, 20)));
        System.out.println(euclideanDistance(0, 0, 20, 20));
    }

}

File: Faltantes o por revisar/EuclideanDistance.py
import math
class Point:
    def __init__(self, x, y):
        self.x = x
        self.y = y
 
    def equals(self, other):
        if (abs(self.x - other.x) < 1e-9 and (abs(self.y - other.y) < 1e-9)):
            return True
        return False
def euclideanDistance1(p1, p2):
    return math.hypot(p1.x - p2.x, p1.y - p2.y);


def euclideanDistance(x1, y1, x2, y2):
    return math.hypot(x2 - x1, y2 - y1)
print(euclideanDistance1(Point(0, 0), Point(20, 20)))
print(euclideanDistance(0, 0, 20, 20))